modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment.
+
+== Run your plug-in locally
+
+.Procedure
+
+. Run `bridge` locally and instruct it to proxy requests directly to your local plug-in asset server (web server hosting the plugin's generated assets): `./bin/bridge -plugins console-demo-plugin=http://localhost:9001/`
```

Comment:
Hey, @opayne1. I'm not sure if you're still working on this, but running bridge this way is no longer necessary. We should follow the steps laid out in the local development section of the template README here:

https://github.com/spadgett/console-plugin-template#option-1-local